js/NavMainComponent.js

```diff
@@ -4,6 +4,7 @@ var $ = require('jquery');
 
 function NavMainComponent ($html, rootWindow) {
     this.$html = $html;
+    this.window = rootWindow;
     this.$window = $(rootWindow);
```

Comment:
Could you please explain why we need both window and $window? What are the limitations of the code. Is it only for the stub in the testing? I don't understand that part

Reply:
`this.$window` already existed in this test but wouldn't work with matchMedia, which required the raw `window`. Looking at it now I can probably just pass `rootWindow` to matchMedia instead so I'll refactor 🤔

Reply:
not going to refactor actually, they're for different things, some functions use the jQuery object, matchMedia needs the raw window object